### ISSUE TITLE
Proxy Error: Unexpected close tag

### ISSUE THREAD

**Issue by ghost** [ddsol/speedtest.net#52]
Hello,

I am trying to use the proxy feature, and I get an unexpected close tag anyway I run it.

I have tried speedtest-net --proxy "http://example.com:80"

I have also tried doing it directly in the code using 

```var test = speedTest({maxTime: 5000, proxy : "http://example.com:80"});```

The error is as follows:

```
Error: Unexpected close tag
Line: 109
Column: 7
Char: >
    at error (/speedtest/node_modules/sax/lib/sax.js:651:10)
    at strictFail (/speedtest/node_modules/sax/lib/sax.js:677:7)
    at closeTag (/speedtest/node_modules/sax/lib/sax.js:871:9)
    at SAXParser.write (/speedtest/node_modules/sax/lib/sax.js:1436:13)
    at Parser.exports.Parser.Parser.parseString (/speedtest/node_modules/xml2js/lib/parser.js:322:31)
    at Parser.parseString (/speedtest/node_modules/xml2js/lib/parser.js:5:59)
    at exports.parseString (/speedtest/node_modules/xml2js/lib/parser.js:354:19)
    at /speedtest/node_modules/speedtest-net/index.js:299:5
    at /speedtest/node_modules/speedtest-net/index.js:107:16
    at IncomingMessage.<anonymous> (/speedtest/node_modules/speedtest-net/index.js:175:7)
```

**Comment by ddsol** [comment]
@devildant

**Comment by devildant** [comment]
Hello,

I use the speedtest with proxy every day and I have no problem, and, this is not an error because of the new proxy option.
it's an xml parse error @ddsol 
@AdamSWheatley your proxy exist ? Is not this a proxy with authentication?

**Comment by AdamSWheatley** [comment]
I use the proxy for other applications and definitely exists lol, It doesn't have any authentication. I assume it's a parse error with the proxy string, the only difference it has than the example provided is it has several dots, so http://test.one.uk.com:80 (not actually it)

**Comment by devildant** [comment]
I do not see how debug that.
can you add the code below at line 299 in node_modules/speedtest-net/index.js (before parseXML(data, function(err, xml)) to see what is returned?
`console.log(data);`

**Comment by ddsol** [comment]
Better error added in #54

**Comment by ddsol** [comment]
Fixed in #82
